### PR TITLE
Fixed checkbox template override

### PR DESCRIPTION
**Pull request by simialbi**
| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️ 
| New feature?  | ❌ 
| Breaks BC?    | ❌ 
| Tests pass?   | ✔️ 
| Fixed issues  | https://github.com/yiisoft/yii2-app-basic/pull/191

### CODE REVIEW

**Comment by samdark** [comment]
How about covering customization of template with a test?

**Comment by samdark** [comment]
Sweet. Thanks!

File: src/ActiveField.php [hunk 1]
```diff
@@ -216,6 +215,9 @@ public function checkbox($options = [], $enclosedByLabel = false)
             Html::removeCssClass($this->labelOptions, $this->horizontalCssClasses['label']);
             Html::addCssClass($this->wrapperOptions, $this->horizontalCssClasses['offset']);
         }
+        if (isset($options['template'])) {
```
Comment (rob006) [inline src/ActiveField.php]: This condition is unnecessary - calling `unset()` on non-existing key does not trigger any error, so from performance perspective it is better call `unset($options['template']);` blindly without any `if`.

File: src/ActiveField.php [hunk 2]
```diff
@@ -248,6 +249,9 @@ public function radio($options = [], $enclosedByLabel = false)
...
```
Comment (rob006) [inline src/ActiveField.php]: Same here.

File: src/ActiveField.php (hunk 1 above)
Comment (simialbi) [reply src/ActiveField.php]: I'll change

**Pull request merged**